src/services/software-modernisation/index.html

```diff
@@ -320,7 +320,7 @@ <h3 class="h4 text-700">{% t software-modernisation.card-4-heading %}</h3>
 <div class=soft-mod__sheen-container>
   {% include compass_teaser.html %}
   {% include soft_mod_webinar_hero_teaser.html %}
-  {% include soft_mod_curated_publications.html %}
+  {% include services_curated_publications.html service-line="softmod" tag="software modernisation" pinned_location="soft-mod" %}
```

Comment:
As mentioned elsewhere, we may be able to include the curated_publications component directly, without the need for any interim includes.

Reply:
This has been done and merged